### PR TITLE
Feature/email verification

### PR DESCRIPTION
**Pull request by dimitrijevic12**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Service/EmailVerificationService.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            string path = Directory.GetParent(Directory.GetCurrentDirectory()).FullName + "\\HealthClinic.DAL\\Html for mail\\verMail1.html";
+            string body = System.IO.File.ReadAllText(path);
+            body = body.Replace("#PatientId#", id.ToString());
+            return new MailMessage(senderAddress, recipientAddress)
+            {
+                Subject = "Verification of HealthClinic account.",
+                //Body = "To verify your HealtClinic account please click on link below.\n" +
+                //        "http://localhost:60198/api/patientuser/" + id
+                //Body = "<h3> To verify your HealtClinic account please click on link below.</h3><a href=\"http://localhost:60198/api/patientuser/" + id + "\"> Link to verify </a>"
```
Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.DAL/Service/EmailVerificationService.cs:45]: Need to delete unnecessary comments

File: PatientWebApplication/PatientWebApplicationTests/EmailVerificationTest.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        [Fact]
+        public void Sends_verification_email()
+        {
+            var mockVerify = new Mock<IEmailVerificationService>();
+            PatientService patientService = new PatientService(CreateStubRepository(), mockVerify.Object);
+
+            PatientUser patient1 = patientService.Create(new PatientDto("Pera2", "Peric", "Male", "1234", "11/11/2000", "1231412", "21312312", "Alergija", "Grad", "email@gmail.com", "pass", false, "Grad2", "Roditelj", "", ""));
+
+            mockVerify.Verify(v => v.SendVerificationMail(new MailAddress(patient1.email), patient1.id), Times.Once);
+            //It.IsAny<MailAddress>(), It.IsAny<int>()
```
Comment (Nemanja0505) [inline PatientWebApplication/PatientWebApplicationTests/EmailVerificationTest.cs:26]: Need to delete this comment

**Review by Nemanja0505 — CHANGES_REQUESTED** [review]
PR is not approved, because code has some unnecessary comments

**Review by Nemanja0505 — APPROVED** [review]
PR is approved. Good practices and principles of clean code are followed.

**Review by Maja0505 — APPROVED** [review]
PR is approved. The task solution is implemented with good Clean code principles.

**Pull request merged**